straxen/get_corrections.py

```diff
@@ -134,52 +117,18 @@ def get_correction_from_cmt(run_id, conf):
                 raise ValueError(f"Could not find a value for {model_type} "
                                  "please check it is implemented in CMT. "
                                  f"for nT = {is_nt}")
+            if model_conf in corrections_w_file:
+                this_file = ' '.join(map(str, correction))
+                return this_file
 
-        if 'samples' in model_type:
+        if 'samples' in model_conf:
             return int(correction)
+
         else:
             return float(correction)
 
     else:
         raise ValueError("Wrong configuration. "
                          "Please use the following format: "
-                         "(model_type->str, model_config->str or number, is_nT->bool) "
+                         "(config->str, model_config->str or number, is_nT->bool) "
                          f"User specify {conf} please modify")
-
-
-@export
-@correction_options
-def get_config_from_cmt(run_id, conf):
-    if isinstance(conf, str) and conf.startswith('https://raw'):
-        # Legacy support for pax files
-        return conf
-    if not isinstance(conf, tuple):
-        raise ValueError("conf must be a tuple")
-    if not len(conf) == 3:
-        raise ValueError("conf must have three elements: "
-                         "the model type, its specific configuration "
-                         "and detector (True = nT)")
-    model_type, model_conf, is_nt = conf
-    if model_type == 'CMT_model':
-        cmt = straxen.CorrectionsManagementServices(is_nt=is_nt)
-        this_file = cmt.get_corrections_config(run_id, model_conf)
-        this_file = ' '.join(map(str, this_file))
-
-    else:
-        raise ValueError(f"Wrong NN configuration, please look at this {conf} "
-                         "and modify it accordingly")
-
-    return this_file
-
-
-def get_elife(run_id, elife_conf):
-    # 1T support for electron lifetimes from a file
-    # Let's remove these functions and only rely on the CMT in the future
-    x = straxen.get_resource(elife_conf, fmt='npy')
-    run_index = np.where(x['run_id'] == int(run_id))[0]
-    if not len(run_index):
-        # Electron lifetime not known: using placeholders
-        e = 623e3
-    else:
-        e = x[run_index[0]]['e_life']
-    return float(e)
```

Comment:
It's probably easier to leave this as it is right, rather than putting this in `get_to_pe`

Reply:
That is not what was done not sure what do you mean, that being said, there is no reason to duplicate functions that do the same, this is something I learn from Daniel